### DISCUSSION TITLE
Several firmware updates for my Lenovo T13 Gen3 AMD showing up under Gnome Software updates ...

### DISCUSSION THREAD

**Discussion by matthewlenz** [fwupd/fwupd#5816] [General]
and they all applied and worked perfectly! Including the fwupd-efi boot shim.  Wow!!!! So cool :D  I normally wouldn't post off-topic stuff like this but this is so seamless I couldn't help myself. Very impressive everyone.

**Comment by hughsie** [comment]
Thanks. :blush:

**Reply by matthewlenz** [reply]
Are there any  high level overviews on how the fwupd-efi boot works and what exactly it does?

**Reply by hughsie** [reply]
Does https://github.com/fwupd/fwupd/blob/main/plugins/uefi-capsule/README.md#update-behavior help?

**Reply by matthewlenz** [reply]
Yep it contains a enough info that I was able to get the gist :D  Thanks!
